locales/nl_NL/pcsx2_Iconized.po

```diff
@@ -1,884 +1,908 @@
-
-#: pcsx2/x86/sVU_zerorec.cpp:367
-msgid ""
-"Out of Memory (sorta): The SuperVU recompiler was unable to reserve the "
-"specific memory ranges required, and will not be available for use.  This is "
-"not a critical error, since the sVU rec is obsolete, and you should use "
-"microVU instead anyway. :)"
-msgstr ""
-"Geheugen vol (soortvan): De SuperVU-recompiler kon de vereiste specifieke "
-"geheugenbereiken niet reserveren, en is niet beschikbaar voor gebruik.  Dit "
-"is geen kritieke fout, omdat de sVU rec verouderd is, en u in plaats daarvan "
-"toch microVU zou moeten gebruiken. :)"
-
-#~ msgid ""
-#~ "Playstation game discs are not supported by PCSX2.  If you want to "
-#~ "emulate PSX games then you'll have to download a PSX-specific emulator, "
-#~ "such as ePSXe or PCSX."
-#~ msgstr ""
-#~ "Playstation-spelschijven worden niet ondersteund door PCSX2.  Als u PSX-"
-#~ "spellen wilt emuleren, moet u een PSX-specifieke emulator downloaden, "
-#~ "zoals ePSXe of PCSX."
-
-#~ msgid ""
-#~ "The Presets apply speed hacks, some recompiler options and some game "
-#~ "fixes known to boost speed.\n"
-#~ "Known important game fixes will be applied automatically.\n"
-#~ "\n"
-#~ "Presets info:\n"
-#~ "1 -     The most accurate emulation but also the slowest.\n"
-#~ "3 --> Tries to balance speed with compatibility.\n"
-#~ "4 -     Some more aggressive hacks.\n"
-#~ "6 -     Too many hacks which will probably slow down most games.\n"
-#~ msgstr ""
-#~ "De profielen passen snelheidshacks, enkele recompileropties en spelfixes "
-#~ "met bekende snelheidverhoging toe.\n"
-#~ "Bekende belangrijke spelfixes worden automatisch toegepast.\n"
-#~ "\n"
-#~ "Profielinfo:\n"
-#~ "1 -     De meest nauwkeurige emulatie, maar ook de langzaamste.\n"
-#~ "3 --> Probeert snelheid te balanceren met compatibiliteit.\n"
-#~ "4 -     Enkele meer agressieve hacks.\n"
-#~ "6 -     Te veel hacks die waarschijnlijk de meeste spellen zullen "
-#~ "vertragen.\n"
-
-#~ msgid ""
-#~ "The Presets apply speed hacks, some recompiler options and some game "
-#~ "fixes known to boost speed.\n"
-#~ "Known important game fixes will be applied automatically.\n"
-#~ "\n"
-#~ "--> Uncheck to modify settings manually (with current preset as base)"
-#~ msgstr ""
-#~ "De profielen passen snelheidshacks, enkele recompileropties en spelfixes "
-#~ "met bekende snelheidverhoging toe.\n"
-#~ "Bekende belangrijke spelfixes worden automatisch toegepast.\n"
-#~ "\n"
-#~ "--> Deselecteer om instellingen handmatig te wijzigen (met huidige "
-#~ "voorinstelling als basis)"
+# Translations for PCSX2 (Iconized).
+# Copyright (C) 2018 PCSX2 Dev Team
+# This file is distributed under the same license as the PCSX2 package.
+# First author: mourix, 2018
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: PCSX2 1.6.0\n"
+"Report-Msgid-Bugs-To: https://github.com/PCSX2/pcsx2/issues\n"
+"POT-Creation-Date: 2020-01-15 22:32+1300\n"
+"PO-Revision-Date: 2020-01-27 00:22+0100\n"
+"Last-Translator: Jos van Mourik\n"

```

Comment:
Maybe you can edit/add yourself here as well since you'll be the latest translator.